### PR TITLE
support for managed disks

### PR DESCRIPTION
**Pull request by snehaamicrosoft**

### CODE REVIEW

**Comment by snehaamicrosoft** [comment]
@sptramer Can you please merge these changes to the master branch! Thanks!
#sign-off

File: azure-migrate/migrate-at-scale-with-site-recovery/asr_startmigration.ps1 [hunk 1]
```diff
@@ -45,15 +47,19 @@ Function ProcessItemImpl($processor, $csvItem, $reportItem) {
...
+        $targetPostFailoverLogStorageAccount = Get-AzureRmStorageAccount `
+            -Name $targetPostFailoverLogStorageAccountName `
+            -ResourceGroupName $targetLogStorageAccountRG
+
+        $targetResourceGroupObj = Get-AzureRmResourceGroup -Name $targetPostFailoverResourceGroup
+        $targetVnetObj = Get-AzureRmVirtualNetwork `
             -Name $targetPostFailoverVNET `
             -ResourceGroupName $targetVNETRG 
-        $targetPolicyMap  =  Get-AzRecoveryServicesAsrProtectionContainerMapping `
+        $targetPolicyMap  =  Get-AzureRmRecoveryServicesAsrProtectionContainerMapping `
```
Comment (sptramer) [inline azure-migrate/migrate-at-scale-with-site-recovery/asr_startmigration.ps1]: Do not use AzureRM; use the Az module. All documentation and sample scripts have moved to Az by the end of Q3.

Comment (snehaamicrosoft) [reply azure-migrate/migrate-at-scale-with-site-recovery/asr_startmigration.ps1]: Thanks @sptramer for the review, have fixed the scripts to use Az module. Please take a look now. #sign-off

**Comment by snehaamicrosoft** [comment]
#sign-off

**Pull request merged**